packages/dataviz/src/components/BarChart/HorizontalBarChart/HorizontalBarChart.component.tsx

```diff
@@ -38,65 +41,65 @@ function HorizontalBarChart({
 		onBarClick,
 		getTooltipContent,
 	);
-
+	// If no height is provided, guess it from content (we don't want the bars to stretch to available height)
+	const contentHeight = height || X_AXIS_TICKS_HEIGHT + data.length * (BAR_GAP + BAR_HEIGHT);
 	return (
-		<ResponsiveContainer width={width} height={height}>
-			<FixedBarSizeWrapper data={data}>
-				<BarChart
-					layout="vertical"
-					className={styles['horizontal-bar-chart']}
-					onMouseMove={onMouseMove}
-					onClick={onClick}
-					onMouseOut={onMouseOut}
-				>
-					<CartesianGrid strokeDasharray="3 3" horizontal={false} />
+		<ResponsiveContainer width={width} height={height || contentHeight}>
```

Comment:
You already have this condition on the `contentHeight` assignment.
```suggestion
		<ResponsiveContainer width={width} height={contentHeight}>
```

Reply:
good catch